x-pack/plugin/ml/qa/native-multi-node-tests/src/test/java/org/elasticsearch/xpack/ml/integration/MlNativeDataFrameAnalyticsIntegTestCase.java

```diff
@@ -170,6 +172,17 @@ protected static DataFrameAnalyticsConfig buildAnalytics(String id, String sourc
             .build();
     }
 
+    protected static DataFrameAnalyticsConfig buildAnalytics(String id, String sourceIndex, String destIndex,
+                                                             @Nullable String resultsField, DataFrameAnalysis analysis,
+                                                             QueryBuilder queryBuilder) throws Exception {
+        return new DataFrameAnalyticsConfig.Builder()
```

Comment:
The other method called `buildAnalytics` (starting in line 165) should call this one, as the new method is more generic.